src/Misc/layoutbin/en-US/strings.json

```diff
@@ -579,6 +585,7 @@
   "UnconfigAutologon": "Removing autologon settings",
   "UnconfigureOSXService": "Unconfigure service first according to https://www.visualstudio.com/en-us/docs/build/admin/agents/v2-osx",
   "UnconfigureServiceDService": "Unconfigure service first according to https://www.visualstudio.com/en-us/docs/build/admin/agents/v2-linux",
+  "UnexpectedParallelCount": "Unsupported parallel count '%s'. Enter a number between 1 and 128.",
```

Comment:
>parallel [](start = 42, length = 8)

parallelism

Reply:
I don't see an issue with current naming and it's also what the existing task is using.